jenkins/job_manager/script/Job.groovy

```diff
@@ -51,13 +51,14 @@ class Job {
     }
 
     def makeName(settingsIndex = null) {
+        def formattedName = branch.name.replace("#", "_no_").replace("/", "_")
```

Comment:
This replacement may lead to name conflicts. I would recommend to change it to `_sl_` or `_slash_`